pkg/ddc/alluxio/utils.go

```diff
@@ -31,6 +31,14 @@ import (
 	"github.com/fluid-cloudnative/fluid/pkg/utils/kubeclient"
 )
 
+func (e *AlluxioEngine) getDataSetFileNum() (string, error) {
+	fileCount, err := e.TotalFileNums()
```

Comment:
TotalFileNums（） has performance issue. Could you add another function to implement.

Reply:
I agree with cheyang`s comment, you can see the performance compare in here, https://github.com/fluid-cloudnative/fluid/issues/299#issuecomment-719902572.

Reply:
okay，have fixed it in new commits